FRW/Modules/Wallet/SideMenu/SideMenuView.swift

```diff
@@ -562,6 +586,28 @@ struct SideContainerView: View {
             }
             .offset(x: vm.isOpen ? screenWidth - SideOffset : 0)
         }
+        .customAlertView(
```

Comment:
There should be a better place for placing alert. 
And it alert shouldn't be configured on SideContainerView.

Reply:
I can't find any root-level view that could be used to host thie alert. The tab view is created here, and there are 3 different variants of it, not using a custom type, but instantiated directly as `TabBarView` — could you suggest me a better place?

Reply:
As mentioned in PM, I've wrapped the alert view in a UIHostingController